ws-nextjs-app/cypress/e2e/languagesPage/index.cy.ts

```diff
@@ -1,29 +1,24 @@
 import { STATIC_PAGE } from '#app/routes/utils/pageTypes';
-import { assertPidginHomepage, assertWSLanguagesPage } from './assertions';
+import {
+  assertWSLanguagesPage,
+  assertWSLanguagesPageURN,
+  assertWSLanguagesPageLocal,
+} from './assertions';
 import runTestsForPage from '../../support/helpers/runTestsForPage';
 
-const homePageTestSuites = [
+const testSuites = [
   {
     path: '/ws/languages',
-    service: 'pidgin',
-    runforEnv: ['test'],
-    tests: [assertPidginHomepage],
-  },
-  {
-    path: '/ws/languages?renderer_env=test',
-    service: 'pidgin',
-    runforEnv: ['test'],
-    tests: [assertPidginHomepage],
+    service: 'ws',
+    runforEnv: ['local'],
+    tests: [assertWSLanguagesPageLocal],
```

Comment:
If you use the fixture data from the WS Languages page, you could also assert on `assertWSLanguagesPageURN`